rwengine/src/script/ScriptFunctions.hpp

```diff
@@ -211,6 +213,26 @@ inline void addObjectToMissionCleanup(const ScriptArguments& args,
         args.getState()->missionObjects.push_back(object);
     }
 }
+
+inline void getClosestNode(const ScriptArguments& args, ScriptVec3& coord, AIGraphNode::NodeType type,
+                           ScriptFloat& xCoord, ScriptFloat& yCoord, ScriptFloat& zCoord) {
+    coord = script::getGround(args, coord);
+    float closest = 2000.f;
```

Comment:
This feels kind of arbitrary.

Reply:
IIRC, original game uses arbitrary value of something like 99999.f too.

Reply:
If it does, we should figure it out (and document it), or choose a large enough value / always accept the first entry to set the baseline. I feel 2000.0f might be too small.

Reply:
I think it should be within game map boundaries. Isn't III map 4000x4000, is it?

Reply:
Wouldn't it make more sense to use a value which is definitely *not* in the game bounds [, but way larger]?
if you are in the center (2000,2000), and you look for objects within a 2000 unit radius in a 4000x4000 square, then you won't be able to find objects near the corners of the game world.

Reply:
Yeah, sorry for misunderstanding. I meant that it should cover map boundaries.

Reply:
This value was found to be 10000.f in @husho 's research.